### PR TITLE
Add improvements

### PR DESCRIPTION
**Pull request by ttytm**
Some more polishing and improvements, split into commits.

### CODE REVIEW

**Review by ttytm — COMMENTED** [review]
Removal of `data/highlight.min.js` is one that i wasn't fully sure about. It doesn't seem to be used anywhere. But it's possible that I don't see the full picture on my end and it's used during doc generation or something.

**Comment by ttytm** [comment]
Included the tailwind change here. This gives broader tailwind support and produces a ~15kb instead of using 3mb style 🥳

To test locally: 
- tailwindcss cli must be installed: https://tailwindcss.com/blog/standalone-cli
- generate the style.css: tailwindcss -i server/data/main.css -o server/style.css --minify

The style file is ignored and if I don't messed up it's generated via the CI action before the ftp upload.

**Comment by hassandraga** [comment]
> Removal of data/highlight.min.js is one that i wasn't fully sure about

I guess I used changed `highlight.min.js` to `prism.js` in the documentation pages a while ago... probably I forgot to remove it from the website. So yeah, it's good that you remove it, I guess.

File: server/index.html [hunk 1]
```diff
@@ -8,92 +8,29 @@
...
               <title>Menu</title>
               <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"></path>
             </svg>
           </button>
         </div>
         <div class="w-full flex-grow lg:flex lg:items-center lg:w-auto hidden lg:block mt-2 lg:mt-0 lg:bg-transparent p-4 lg:p-0 z-20 bg-gray-100" id="nav-content">
-          <ul id="menu" class="text-black lg:text-white lg:text-lg lg:flex justify-end flex-1 items-center my-4">
+          <ul id="menu" class="text-black lg:text-white [.scrolled_&]:text-gray-900 lg:text-lg lg:flex justify-end flex-1 items-center my-4">
             <li class="mr-3">
-              <a class="inline-block py-2 px-4 no-underline" href="https://webui.me/">Home</a>
```
[the remark this answers is not in the document]
Comment (hassandraga) [reply server/index.html:96]: I fix it

**Review by hassandraga — APPROVED** [review]

**Comment by ttytm** [comment]
I just see at a wider window that with the tailwind cli we have new xl width for the `container` class:

```
@media (min-width: 1536px)
.container {
```

before it ended at 1280

the header is to wide now on wide screens. I quickly update this.

File: server/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (ttytm) [reply server/index.html:96]: Okay, thanks @hassandraga 👍 

I added it so the items in the nav all scroll (despite dev as it's a link to a new page). 
On the left side, the logo, still had the link. But it's just as good. And maybe the link also scrolls when it's live on the server.

**Pull request merged**